textile/chat-features.textile

```diff
@@ -368,10 +375,11 @@ Typing Indicators allow chat room users to indicate to others that they are typi
 ** @(CHA-T2b)@ This specification point has been removed.
 *** @(CHA-T2b1)@ This specification point has been removed.
 *** @(CHA-T2b2)@ This specification point has been removed.
-** @(CHA-T2c)@ @[Testable]@ If the room status is @ATTACHING@, then the @get@ call shall wait for the attach operation to succeed and then call the underlying presence @get@ operation. It must throw an error if the attach fails.
+** @(CHA-T2c)@ @[Testable]@ If the room status is @Attaching@, then the operation shall enter a @CHA-RL9@ pending room status operation. If this succeeds (i.e the room becomes @ATTACHED@), the library must invoke the underlying @presence.get()@ call.
```

Comment:
```suggestion
** @(CHA-T2c)@ @[Testable]@ If the room status is @Attaching@, then the operation should wait as per @CHA-RL9a@, once RoomStatus is @Attached@ as per @CHA-RL9b@ then the library must invoke the underlying @presence.get()@ call, otherwise throw error as per @CHA-RL9c@
```

Reply:
Will mark this as resolved and continue discussion on the one I've left open